weave-js/src/components/Form/SelectMultiple.tsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * Customized version of react-select for selecting multiple options.
+ */
+
+import React, {MouseEventHandler} from 'react';
+import {
+  components,
+  MultiValueGenericProps,
+  MultiValueProps,
+  Props as ReactSelectProps,
+} from 'react-select';
+import {
+  SortableContainer,
+  SortableContainerProps,
+  SortableElement,
+  SortableHandle,
+  SortEndHandler,
+} from 'react-sortable-hoc';
+
+import {AdditionalProps, Select} from './Select';
+
+function arrayMove<T>(array: readonly T[], from: number, to: number) {
```

Comment:
could you add a comment on what this function does?

Reply:
oh i see that its just moved from the example, you can either add the comment or maybe just add the example link